Analytics/EntityStoreTools/EntityStoreMetadataExporter/Utils/ColorConsole.cs

```diff
@@ -0,0 +1,41 @@
+﻿// ------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------------
+
+namespace EntityStoreMetadataExporter.Utils
+{
+    using System;
+
+    /// <summary>
+    /// Utility class to print colored console output.
+    /// </summary>
+    public static class ColorConsole
```

Comment:
This class is also duplicated... Is there a way to put all of them in a common package and reuse them across the tools?